components/Cards/Card.tsx

```diff
@@ -8,11 +8,11 @@ const Card: FC<{ data: IData }> = (props) => {
   const { name, description, url } = data
 
   return (
-    <article className="z-10 h-full w-full rounded-3xl border border-dashed border-violet-500 bg-gray-100 shadow-lg dark:bg-gray-900 dark:text-gray-300 dark:shadow-sm">
+    <article className="z-10 h-full w-full rounded-3xl border border-dashed border-violet-500 dark:border-violet-400 bg-gray-100 shadow-lg dark:bg-gray-900 dark:text-gray-300 dark:shadow-sm">
```

Comment:
Color contrast of `violet-500` isn't enough in dark mode. Not enough color contrast means folks with limited perception, such as screen glare, low quality monitors, or personal vision impairments may not be able to read the text.

https://webaim.org/resources/contrastchecker/

Reply:
Great, we would need to change it for other components  like the _popup_ as well